src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -65,6 +66,11 @@ namespace subgraph
 
 variables {V : Type u} {G : simple_graph V}
 
+/-- `H.support` is the set of vertices that form edges in the subgraph `H`. -/
+def support (H : subgraph G) : set V := rel.dom H.adj
+
+lemma mem_support (H : subgraph G) {v : V} : v ∈ H.support ↔ ∃ w, H.adj v w := iff.rfl
+
```

Comment:
Let's move these down to right above `neighbor_set`, paralleling their location in `basic.lean`.

You should be able to also prove
```lean
lemma support_mono {H H' : subgraph G} (h : H ≤ H') : H.support ⊆ H'.support :=
rel.dom_mono h -- just guessing here, haven't tested
```